src/test/java/uk/gov/companieshouse/search/api/interceptor/UserAuthorisationInterceptorTest.java

```diff
@@ -18,7 +18,7 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.when;
-import static uk.gov.companieshouse.search.util.EricHeaderHelper.ERIC_IDENTITY_TYPE;
+import static uk.gov.companieshouse.search.api.util.EricHeaderHelper.ERIC_IDENTITY_TYPE;
```

Comment:
static imports should be before non-static imports as specified https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing